src/shared/icons/LeftArrow.tsx

```diff
@@ -0,0 +1,3 @@
+export const LeftArrow = (): JSX.Element => <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it will better to put our svg code in another folder. For examle, we can create all aour svg files and components with svg inside src/shared/assets/svg/ folder.